modular_skyrat/code/modules/mob/living/carbon/human/species_types/speciessnowflake.dm

```diff
@@ -0,0 +1,95 @@
+
+/datum/species/dullahan
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Not much is known about the dullahan, but they sure know how to make someone lose their minds."
+
+/datum/species/dwarf
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Dwarves are very similar to humans in biology, but they seem to have completely split from them as a species."
+
+/datum/species/mammal
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Anthropomorphs encompass a lot of the know sapient species by SolGov, and their backgrounds always vary wildly."
+
+/datum/species/jelly
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Slimepeople are the result of scientifical experimentation on slimes, which NanoTrasen has heavily invested in. At the current day, most slimepeople are simply consumers of brands of slime genemods sold by NanoTrasen."
+
+/datum/species/plasmaman
+	screamsounds = list('modular_citadel/sound/voice/scream_skeleton.ogg')
+	fluff_desc = "Plasmamen are the result of catastrophic experimentation with plasma. They are very fragile and in most circumstances require external support to survive, and for this reason are considered most of the time simply cheap workforce."
+
+/datum/species/pod
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Podpeople are simply plantified clones of other sapient species - No one knows where podplants were found for sure, but podpeople have since formed splinter groups about glorifying and preserving nature, coexisting with advanced technology."
+
+/datum/species/shadow
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Shadowpeople are considered ashlanders that were somehow cursed by the necropolis. They are doomed to live in the dark, as light burns their flesh away."
+
+/datum/species/synthliz
+	screamsounds = list('modular_citadel/sound/voice/scream_silicon.ogg')
+	fluff_desc = "Synthetic lizardpeople are simply a variant of IPCs made for the sake of looking more appealing, but suffer with the same problems that IPCs do."
+
+/datum/species/vampire
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Vampires are cursed humans, doomed to feast on blood to survive."
+
+/datum/species/vox
+	screamsounds = list('modular_skyrat/sound/emotes/voxscream.ogg')
+	fluff_desc = "Vox are an avian species found around the frontier. They are seem as terrible raiders by most other species, and suffer prejudice for it."
```

Comment:
"Vox are an avian-like species found mostly in the frontier of civilized space or some distance outside of it. Their original culture and planets are long lost to history, replaced nowadays by roaming bands of raiders, traders, and refugees seeking a place to call home."